django/contrib/auth/apps.py

```diff
@@ -18,8 +19,10 @@ def ready(self):
             create_permissions,
             dispatch_uid="django.contrib.auth.management.create_permissions"
         )
-        if hasattr(get_user_model(), 'last_login'):
+        last_login_field = getattr(get_user_model(), 'last_login', None)
+        if isinstance(last_login_field, DeferredAttribute):
             from .models import update_last_login
             user_logged_in.connect(update_last_login, dispatch_uid='update_last_login')
+
```

Comment:
Please chop this unrelated blank line.

Reply:
Hey @felixxm. I looked at both of these and thought them OK because they aided readability. (Just a small visual separator between the blocks). 

Is your objection that to the lines themselves, or that they're unrelated to the change per se? If the latter would a separate commit be better? 

(Thanks!)

Reply:
I wouldn't add the extra lines for both reasons. Not to say that mistakes aren't made, but generally if we merged a previous patch without them (at least recently when code style is more standardized than in the early days), then blank line changes probably aren't needed. Certainly putting unrelated blank line additions in a patch doesn't help keep things clean and I don't think the blank lines are so important that they deserve a commit.

Reply:
Hmmm... Not sure I entirely agree. (Which is OK.)

In general it seems to leave no space for the little changes which over-time add up to more readable code: can't add to a change commit, because they're unrelated, can't add as a separate commit because (yes, you're absolutely right) individually they don't add up to much.

In particular the whitespacing in the (other) `test_signals` block isn't even consistent within the single test case — I thought the extra line there just such one of these insignificant cases that is better afterwards (since the spacing is then consistent).  

Anyhow, no a biggie here.

Reply:
The reason there's no whitespace after `with self.assertRaises(FieldDoesNotExist):` is because that's logically connected to the with statement that follows. I added `self.assertIsNone(UserWithDisabledLastLoginField().last_login)` to the new test which serves a similar purpose.

Reply:
OK. Makes sense. Ta!